dev/dll/Microsoft.UI.Xaml.vcxproj

```diff
@@ -210,12 +211,13 @@
       <AdditionalIncludeDirectories>$(MSBuildProjectDirectory)\..;$(MSBuildProjectDirectory)\..\..\idl;$(MiniWindowsSDKIncludePath)</AdditionalIncludeDirectories>
       <Enumclass>false</Enumclass>
       <PreprocessorDefinitions Condition="'$(Configuration)'=='Debug'">%(PreprocessorDefinitions);DEBUG;DBG</PreprocessorDefinitions>
+      <PreprocessorDefinitions Condition="'$(Platform)'=='Arm'">%(PreprocessorDefinitions);ARM32</PreprocessorDefinitions>
     </Midl>
     <Link>
       <ModuleDefinitionFile>Microsoft.UI.Xaml.def</ModuleDefinitionFile>
       <GenerateDebugInformation Condition="'$(Configuration)'=='Debug'">$(GenerateDebugInformation)</GenerateDebugInformation>
       <GenerateDebugInformation Condition="'$(Configuration)'=='Release'">DebugFull</GenerateDebugInformation>
-      <AdditionalDependencies>mincore.lib;dxguid.lib;%(AdditionalDependencies)</AdditionalDependencies>
+      <AdditionalDependencies>mincore.lib;dxguid.lib;user32.lib;%(AdditionalDependencies)</AdditionalDependencies>
```

Comment:
is this right? shouldn't this get the import via windowsapp.lib or similar?

Reply:
Oh, good catch. I need to find a WACK-compliant way to import the USER APIs.